src/status_im2/subs/wallet_2/networks.cljs

```diff
@@ -0,0 +1,33 @@
+(ns status-im2.subs.wallet-2.networks
```

Comment:
happy to adjust these namespaces 👍

Reply:
Since it's inside the `status-im2`, we can drop the suffix `2` in the wallet.